shared/schemas/workflow-options-output-schema.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Workflow Results",
+    "description": "Results of a workflow execution",
+    "type": "object",
+    "properties": {
+        "completedWith": {
+            "description": "The state of workflow completion.",
+            "type": "string",
+            "enum": [
+                "error",
+                "success"
+            ]
+        },
+        "description": {
+            "description": "High-level summary of the current status, free-form text, human readable.",
+            "type": "string"
+        },
+        "nextWorkflows": {
```

Comment:
do we have a use case for more than one workflow? i suggest limiting the implementation to the current needs.